.github/workflows/build_samples.yml

```diff
@@ -0,0 +1,74 @@
+name: Build Samples
+
+on:
+  push:
+    branches:
+      - main
+      - reorganize_fcore
+  pull_request:
+    branches:
+      - '*'
+  schedule:
+    - cron: "0 0 * * 1"
+env:
+  test_roots: |
+    app/samples/littlefs
+    app/samples/sensor
+    app/samples/shield_blinky
+    app/samples/shield_mcp3561r
```

Comment:
Can remove ones pre F-Core

Reply:
I was thinking it might be nice to keep them. I didnt include the sensor_log sample since the new f core data logging library that is coming soon replaces that. But i think for basic stuff we should know when it builds or breaks with updates bc if we say hey new people, go look at this sample or something we should know when it breaks with updates/board changes and stuff.